ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/424__model_serving_cluster_setting.robot

```diff
@@ -0,0 +1,99 @@
+*** Settings ***
+Documentation    Test cases that verify High Availability for ModelServing CLuster Setting
+Resource         ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Resource         ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Suite Setup      Sute Setup
+Suite Teardown   Sute Tierdown
+Test Setup       Tst Setup
+Test Tags         modelservingsetting  ODS-2574  Tier1  Sanity
+
+*** Variables ***
+${project_tittel}=  BLANKPROJ
+
+*** Keywords ***
+Sute Setup
+    [Documentation]    Opens the Dashboard Settings and Create DS project
+    Set Library Search Order    SeleniumLibrary
+    Launch Dashboard   ocp_user_name=${TEST_USER.USERNAME}  ocp_user_pw=${TEST_USER.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}
+    ...    browser=${BROWSER.NAME}  browser_options=${BROWSER.OPTIONS}
+    ...    expected_page=${NONE}    wait_for_cards=${FALSE}
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${project_tittel}    description=test project
+
+Tst Setup
+    [Documentation]    Opens the Dashboard Settings Deselect all the model serving.
+    Open Dashboard Settings    settings_page=Cluster settings
+    Reload Page
```

Comment:
why?

Reply:
sometime it was taking long time to load we can try removing it  if it works in CI ill remove it